pkgs/by-name/py/pyrefly/package.nix

```diff
@@ -0,0 +1,49 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+let
+  pname = "pyrefly";
+  version = "0.17.1";
+in
+rustPlatform.buildRustPackage {
+  inherit pname version;
```

Comment:
buildRustPackage now supports the `finalAttrs` pattern too, so using a let-in here shouldn't be necessary